applications/nrf_desktop/description.rst

```diff
@@ -1131,11 +1131,6 @@ Building information
 Use the configuration with the ``release`` file suffix for the HID report rate measurement.
 Debug features, such as logging or assertions, decrease the application performance.
 
-.. note::
-   You can run the application code from RAM instead of NVM to improve performance as code execution from the RAM is generally faster than from the NVM.
```

Comment:
Actually, this file should not be touched as the note is already added here- https://github.com/nrfconnect/sdk-nrf/blob/v3.2-branch/applications/nrf_desktop/description.rst?plain=1#L1135. Seems like the note will get removed if this PR is merged. Can this be fixed.